src/bcftools/bcftools_annotate/config.vsh.yaml

```diff
@@ -0,0 +1,251 @@
+        required: true
+         
+  - name: Options
+    description: | 
+      For examples on how to use use bcftools annotate see http://samtools.github.io/bcftools/howtos/annotate.html.
+      For more details on the options see https://samtools.github.io/bcftools/bcftools.html#annotate.
+    arguments:
+      
+      - name: --annotations
+        alternatives: --a
+        type: file
+        description: | 
+          VCF file or tabix-indexed FILE with annotations: CHR\tPOS[\tVALUE]+ . 
+
+      - name: --columns
+        alternatives: --c
+        type: string
+        description: | 
+          List of columns in the annotation file, e.g. CHROM,POS,REF,ALT,-,INFO/TAG. 
+          See man page for details.
+
+      - name: --columns_file
+        alternatives: --C
+        type: file
+        description: | 
+          Read -c columns from FILE, one name per row, with optional --merge_logic TYPE: NAME[ TYPE].
+
+      - name: --exclude
+        alternatives: --e
+        type: string
+        description: | 
+          Exclude sites for which the expression is true.
+          See https://samtools.github.io/bcftools/bcftools.html#expressions for details.
+
+      - name: --force
+        type: boolean_true
+        description: | 
+          continue even when parsing errors, such as undefined tags, are encountered. 
+          Note this can be an unsafe operation and can result in corrupted BCF files. 
+          If this option is used, make sure to sanity check the result thoroughly.
+
+      - name: --header_line
+        alternatives: --H
+        type: string
+        description: | 
+          Header line which should be appended to the VCF header, can be given multiple times.
+
+      - name: --header_lines
+        alternatives: --h
+        type: file
+        description: | 
+          File with header lines to append to the VCF header.
+          For example:
+            ##INFO=<ID=NUMERIC_TAG,Number=1,Type=Integer,Description="Example header line">
+            ##INFO=<ID=STRING_TAG,Number=1,Type=String,Description="Yet another header line">
+
+      - name: --set_id
+        alternatives: --I
+        type: string
+        description: | 
+          Set ID column using a `bcftools query`-like expression, see man page for details.
+
+      - name: --include
+        type: string
+        description: | 
+          Select sites for which the expression is true.
+          See https://samtools.github.io/bcftools/bcftools.html#expressions for details.
+      
+      - name: --keep_sites
+        alternatives: --k
+        type: boolean_true
+        description: | 
+          Leave --include/--exclude sites unchanged instead of discarding them.
+
+      - name: --merge_logic
+        alternatives: --l
+        type: string
+        description: | 
+          When multiple regions overlap a single record, this option defines how to treat multiple annotation values.
+          See man page for more details.
```

Comment:
From the docs it seems there is a limited number of choices for merge logic. If so, could you ad choices here?

Reply:
Because the format is tag:first or tag:append ... etc, and there are multiple tags in a VCF file (plus I think you can create tags as well) I feel like the list of choices would be to big. Thus I think 'choices' is not realy applicable here.